package.json

```diff
@@ -32,15 +32,17 @@
     "envelope-spec": "^1.1.0",
     "lodash.get": "^4.4.2",
     "lodash.set": "^4.3.2",
-    "private-group-spec": "^1.2.0",
+    "private-group-spec": "Powersource/private-group-spec#ssb-uris",
     "pull-async": "^1.0.0",
     "pull-paramap": "^1.2.2",
     "pull-stream": "^3.6.14",
-    "ssb-bfe": "^3.6.0",
-    "ssb-box2": "^3.0.0",
+    "ssb-bfe": "^3.7.0",
+    "ssb-box2": "^3.0.1",
     "ssb-crut": "^4.6.1",
     "ssb-meta-feeds": "~0.30.0",
-    "ssb-private-group-keys": "^1.0.0"
+    "ssb-private-group-keys": "^1.0.0",
+    "ssb-ref": "^2.16.0",
+    "ssb-uri2": "^2.4.0"
```

Comment:
Need to update after https://github.com/ssbc/ssb-uri2/pull/19

Would use the branch already but i don't know how to make ts play nice